src/rules/length.ts

```diff
@@ -6,14 +6,14 @@ const validate = (value: string | number | string[], { length }: Record<string,
     return false;
   }
 
-  if (typeof value === 'number') {
-    value = String(value);
-  }
-
   if (typeof value === 'string') {
     value = Array.from(value);
```

Comment:
Still, there is a direct call of `Array.from` where as the `toArray()` at `/src/utils/collections.ts` has a polyfill fror this.
On the other hand, `toArray()` will not work here, because it will broke #2876 again.

@logaretm are you aware of that?

Reply:
you are right, better to count slightly off, than breaking on browsers without `Array.from`